src/contexts/Drive/Drive.context.tsx

```diff
@@ -46,6 +48,51 @@ interface DriveContextProviderProps {
 const logger = new BaseLogger({
   tag: 'DRIVE_CONTEXT',
 });
+
+export const getModifiedDriveItems = async () => {
+  const lastUpdatedAt = (await asyncStorageService.getItem(AsyncStorageKey.LastUpdatedAt)) || new Date().toISOString();
+
+  const [modifiedFiles, modifiedFolders] = await Promise.all([
+    driveFileService.getModifiedFiles({
+      updatedAt: lastUpdatedAt,
+      status: 'ALL',
+    }),
+    driveFolderService.getModifiedFolders({
+      updatedAt: lastUpdatedAt,
+      status: 'ALL',
+    }),
+  ]).catch((error) => {
+    errorService.reportError(error);
+    return [[], []];
+  });
+
+  if (!modifiedFiles || !modifiedFolders || (!modifiedFiles.length && !modifiedFolders.length)) return;
+
+  const modifiedFilesIds = modifiedFiles.map((file) => file.id);
+  const modifiedFoldersIds = modifiedFolders.map((folder) => folder.id);
+
+  modifiedFilesIds.forEach(async (fileId) => {
+    await driveLocalDB.deleteItem({ id: fileId });
+  });
+
+  modifiedFoldersIds.forEach(async (folderId) => {
+    await driveLocalDB.deleteFolderRecord(folderId);
+  });
+
+  // Get the last updatedAt date from the modified files and folders
+  let lastUpdatedAtFromModifiedFiles;
+  let lastUpdatedAtFromModifiedFolders;
+
+  if (modifiedFiles.length) lastUpdatedAtFromModifiedFiles = _.maxBy(modifiedFiles, 'updatedAt')?.updatedAt;
```

Comment:
Should I remove this conditional since we check if the arrays are not empty before? @PixoDev

Reply:
nope, 

`if (!modifiedFiles.length && !modifiedFolders.length) return;`

Here you are checking that both, files and folders has no length, meaning that if one of them has length, this will not be true, so you'll need to check in case some of the arrays has no length